metomi/rosie/ws.py

```diff
@@ -471,7 +471,19 @@ def parse_cli(*args, **kwargs):
         None:
             bare command, requesting to print server status
     """
-    opt_parser = RoseOptionParser()
+    opt_parser = RoseOptionParser(
+        description='''
+Start/stop ad-hoc Rosie suite discovery web service server.
```

Comment:
Again have't been able to preview this due to disabling but assuming this is in place for reimplementation.